src/blocks/Biometrics/index.native.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+import React from "react";
+import { Button } from "react-native";
+import {
+  supportsBiometrics,
+  authWithBiometrics,
+  withSupportedState,
+  withLifecycle,
+} from "./utils";
+
+export const UnwrappedAuth = ({
```

Comment:
Or rather than that, `TouchId` or `BiometricApprovalButton` or something telling like that